src/commands/createFunction/openAPISteps/OpenAPICreateStep.ts

```diff
@@ -0,0 +1,114 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { OpenDialogOptions, ProgressLocation, Uri, window, workspace } from "vscode";
+import { AzureWizardExecuteStep, AzureWizardPromptStep, DialogResponses, IActionContext } from 'vscode-azureextensionui';
+import { ProjectLanguage } from "../../../constants";
+import { ext } from "../../../extensionVariables";
+import { localize } from "../../../localize";
+import { cpUtils } from "../../../utils/cpUtils";
+import { nonNullProp } from '../../../utils/nonNull';
+import { openUrl } from '../../../utils/openUrl';
+import { IJavaProjectWizardContext } from '../../createNewProject/javaSteps/IJavaProjectWizardContext';
+import { IDotnetFunctionWizardContext } from '../dotnetSteps/IDotnetFunctionWizardContext';
+import { IFunctionWizardContext } from "../IFunctionWizardContext";
+
+export class OpenAPICreateStep extends AzureWizardExecuteStep<IFunctionWizardContext> {
+    public priority: number = 220;
+
+    public static async createStep(context: IActionContext): Promise<OpenAPICreateStep> {
+        await validateAutorestInstalled(context);
+        return new OpenAPICreateStep();
+    }
+
+    public async execute(wizardContext: IFunctionWizardContext & IJavaProjectWizardContext & IDotnetFunctionWizardContext): Promise<void> {
+        // const uris: Uri[] = await this.askDocument();
+        const uris: Uri[] = nonNullProp(wizardContext, 'openApiSpecificationFile');
+        const uri: Uri = uris[0];
+        const args: string[] = [];
+
+        args.push(`--input-file:${cpUtils.wrapArgInQuotes(uri.fsPath)} --version:3.0.6320`);
```

Comment:
I'm still confused about hard-coding this version. When can we remove it - before this PR is merged? And what happens if the users doesn't have the latest version installed on their machine - will it fail?

Reply:
The `--version` argument was mainly as 6320 hadn't been deployed yet. But with [@autorest/core@3.0.6320](https://www.npmjs.com/package/@autorest/core) now deployed. It is not needed anymore now. Removing it now.